website/docs/explorer/configuration.mdx

```diff
@@ -14,15 +14,15 @@ This page helps you to understand the options available to configure Explorer
 
 ## Prerequisites
 Before using Explorer, please ensure that:
-- You have Weave Gitops Enterprise [v0.23.0](../../releases)
+- You have Weave Gitops Enterprise [v0.23.0](../enterprise/getting-started/releases-enterprise.mdx)
```

Comment:
This link currently loads the releases page which at this time has version 0.24.0 at the top. This can be confusing for a user. Perhaps we can add that 0.23.0 is the earlier release that this feature exists in (can go in a follow-up PR).